### ISSUE TITLE
United error response format for management APIs (legacy and v2.0 APIs)

### ISSUE THREAD

**Issue by heww** [goharbor/harbor#10546]
Use the OCI APIs error format or management APIs error format ?

OCI APIs error format
```json
{
    "errors": [
        { "code": "UNKNOW", "message": "Blob unknow" }
    ]
}
```

Managment APIs error format
```json
{
    "code": "BAD_REQUEST", 
    "message": "Name must be unique" 
}
```

**Comment by ywk253100** [comment]
We should estimate the effort to update the legacy APIs to use the error array format. If it is huge effort, I suggest use the single error format.

Besides, the legacy API uses the error with integer error code, we should update it as string

**Comment by ywk253100** [comment]
Fixed by #11070
